nuntium/user_section/views.py

```diff
@@ -414,3 +414,50 @@ class WriteItDeleteView(WriteItInstanceOwnerMixin, DeleteView):
     def get_success_url(self):
         url = reverse('your-instances')
         return url
+
+
+class StatsPerInstance():
+    def __init__(self, writeitinstance=None):
+        self.writeitinstance = writeitinstance
+
+    def get_stats(self):
+        return [
+            ('amount_of_messages', self.amount_of_messages),
+            ('amount_of_public_messages', self.amount_of_public_messages),
+            ('amount_of_private_messages', self.amount_of_private_messages),
+            ('public_messages_with_answers', self.public_messages_with_answers),
+        ]
+
+    @property
+    def amount_of_messages(self):
+        return self.writeitinstance.message_set.count()
+
+    @property
+    def amount_of_public_messages(self):
+        return self.writeitinstance.message_set.filter(public=True).count()
```

Comment:
https://github.com/ciudadanointeligente/write-it/commit/30f4214efcaa8f3e78800f8955cdf6f312a0d45b adds a Public and Confirmed messages.